### ISSUE TITLE
Android triggers an orientations change event when device laid down flat

### ISSUE THREAD

**Issue by trobbertze** [wonday/react-native-orientation-locker#77]
Hi

In Android when you lay your device down flat on a table an orientation change event is fired.

This is not problematic per se, but the issue is that the orientation event always fires with value 'PORTRAIT', even though the phone was in landscape orientation when you laid it down.

This does not happen on IOS.

I am on version 1.1.5 with RN - 0.59.9

**Comment by coder89** [comment]
Very likely it's your devices' specific issue. What's your HW? Works fine on my LG v30.

**Comment by trobbertze** [comment]
I don't think so, we tested on multiple devices over here and the results were consistent.

I don't have my device on me at the moment, its a Samsung, will have to get back to you with the exact model name.  But like I said we saw this behaviour across many different devices.

**Comment by JavanPoirier** [comment]
I have noticed a similar issue on my S8. The orientation event listener is being fired without the actual screen making an orientation change. Slight angling of the device alone will emit the event. This causes some of my UI which is dependant on this listener to change regardless of actual screen orientation change. This might be solved by using the non-device listener however, with the screen locked to any landscape mode (as the documentation mentions) that will not work. Should this continue to be an issue I may tackle this.

**Comment by shubham696** [comment]
@trobbertze did you get any solution for rotation problem?

**Comment by uqpgeor2** [comment]
@JavanPoirier I am also experiencing this issue with the slight angling triggering orientation change. Do you have any plans to address the problem?
